indexing-hadoop/src/test/resources/druid.test.data.tsv

```diff
@@ -0,0 +1,82 @@
+2011-04-10T00:00:00.000Z	spot	automotive	preferred	apreferred	113.221448
```

Comment:
If the only difference between existing file and this one is having duplicate rows, can we make the difference clear in file name ? e.g test_data_with_duplicate_rows.tsv